Makefile

```diff
@@ -26,3 +26,7 @@ add-dep:
 .PHONY: lint
 lint:
 	golangci-lint run
+
+.PHONY: lint
+run:
+	go run ./cmd/baton-snyk
```

Comment:
Other connectors don't have a `make run` rule. I don't think this should be unique.